src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -0,0 +1,253 @@
+
+            <!-- more actions menu -->
+            <span>
+              <v-menu offset-y>
+                <template v-slot:activator="{ on }">
+                  <v-btn text small color="primary" class="actions__more-actions__btn" v-on="on">
+                    <v-icon>mdi-menu-down</v-icon>
+                  </v-btn>
+                </template>
+                <v-list class="actions__more-actions">
+                  <v-list-item @click="confirmRemove(index)">
+                    <v-list-item-title><v-icon>mdi-delete</v-icon>Remove</v-list-item-title>
+                  </v-list-item>
+                </v-list>
+              </v-menu>
+            </span>
+          </div>
+        </v-col>
+      </v-row>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Prop, Mixins, Emit } from 'vue-property-decorator'
+
+// Components
+import BaseAddress from 'sbc-common-components/src/components/BaseAddress.vue'
+
+// Dialogs
+import { ConfirmRemoveDialog } from '@/components/dialogs'
+
+// Mixins
+import CommonMixin from '@/mixins/common-mixin'
+
+// Interfaces
+import { OrgPersonIF } from '@/interfaces'
+
+@Component({
+  components: {
+    BaseAddress,
+    ConfirmRemoveDialog
+  }
+})
+export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
+  // Store Properties
+  @Prop({ default: '' })
+  private personList: Array<OrgPersonIF>
+
+  @Prop({ default: false })
+  private showErrorSummary: boolean
+
+  @Prop({ default: false })
+  private isSummary: boolean
+
+  // Local Properties
+  readonly tableHeaders: Array<string> = ['Name', 'Mailing Address', 'Delivery Address', 'Roles']
+  private dialog: boolean = false
+  private personId: number
+
+  /**
+   * Determine if Corporation/Firm or Person.
+   * @param filing The filing body which contains the name/title.
+   * @returns The appropriate Corporation or Person name.
+   */
+  private formatName (filing: any): string {
+    return filing.orgName ? filing.orgName
+      : `${filing.firstName} ${filing.middleName || ''} ${filing.lastName}`
+  }
+
+  /**
+   * Identify and confirm the removal of a person/org through dialog.
+   * @param index The active index which is subject to removal.
+   */
+  confirmRemove (index: number): void {
+    this.personId = index
+    this.dialog = true
+  }
+
+  /**
+   * Emit an index and event to the parent to handle removal.
+   * @param index The active index which is subject to removal.
+   */
+  @Emit('removePerson')
+  private emitRemovePerson (index: number): void {
+    this.dialog = false
+  }
+
+  /**
+   * Emit an index and event to the parent to handle editing.
+   * @param index The active index which is subject to change.
+   */
+  @Emit('editPerson')
+  private emitPersonInfo (index: number): void {}
+}
+</script>
+
+<style lang="scss" scoped>
+#people-roles {
+  margin-top: 1rem;
+}
+
+.people-roles-summary-header {
+  display: flex;
+  background-color: rgba(1, 51, 102, 0.15);
```

Comment:
Is "$gray1" a suitable color variable that could be used here instead. (If yes, consider replacing it globally.)

Reply:
fixed!